tests/test.rs

```diff
@@ -9,8 +10,8 @@ fn single_encoding() -> Result<(), Error> {
     let mut headers = HeaderMap::new();
     headers.insert(ACCEPT_ENCODING, HeaderValue::from_str("gzip")?);
 
-    let encoding = accept_encoding::parse(&headers)?;
-    assert!(encoding.is_gzip());
+    let encoding = accept_encoding::parse(&headers)?.expect("Something went wrong");
```

Comment:
Could we elaborate on the error message here, or use `.unwrap()` instead?

Reply:
Will opt to just use unwrap instead, since what encoding is should already be known and never fail :sweat_smile: